src/raster/r.random.walk/Makefile

```diff
@@ -0,0 +1,7 @@
+MODULE_TOPDIR = ../..
+
+PGM = r.random.walk
+
+include $(MODULE_TOPDIR)/include/Make/Script.make
+
+default: script
```

Comment:
Line ending is missing here. It should be unix-like line endings, i.e., LF at the end of each line including the last line (they are line endings, not line separators in unix world). Not sure if modern make actually requires that or not, but I noticed because GitHub highlighted the issue.

Reply:
This should be okay now.